render/render.js

```diff
@@ -597,6 +604,9 @@ module.exports = function($window) {
 	}
 
 	function render(dom, vnodes) {
+		// granular update through render(vnode)
+		if (dom.tag && dom.dom.vnodes) render(dom.dom, Object.assign({}, dom))
```

Comment:
Hmm, your approach looks neat. Adding `force` option to `updateNodes()` is still debatable as per your conversation with @pygy